spec/models/user_spec.rb

```diff
@@ -9,9 +9,43 @@
 
       context 'weekday' do
         it 'sends reminders to users without entries from today' do
-          expect(Reminder).to receive(:send_to).twice.and_return(double 'Mailer', deliver: true)
+          expect(Reminder).to receive(:send_to).twice
+            .and_return(double 'Mailer', deliver: true)
 
-          travel_to Time.new(2015, 07, 17, 0, 0, 0) do
+          travel_to Time.new(2016, 03, 23, 0, 0, 0) do
+            User.send_reminders
+          end
+        end
+      end
+
+      context 'holiday in argentina' do
+        before do
+          allow(ENV).to receive(:[]).with("COUNTRY_CODE").and_return("ar")
```

Comment:
@schmierkov Please use https://relishapp.com/rspec/rspec-mocks/v/2-11/docs/stubbing-constants to stub this constant.